src/main/java/org/synyx/urlaubsverwaltung/person/extension/PersonEventHandlerExtension.java

```diff
@@ -0,0 +1,70 @@
+package org.synyx.urlaubsverwaltung.person.extension;
+
+import de.focus_shift.urlaubsverwaltung.extension.api.person.PersonCreatedEventDTO;
+import de.focus_shift.urlaubsverwaltung.extension.api.person.PersonDeletedEventDTO;
+import de.focus_shift.urlaubsverwaltung.extension.api.person.PersonDisabledEventDTO;
+import de.focus_shift.urlaubsverwaltung.extension.api.person.PersonUpdatedEventDTO;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.ApplicationEventPublisher;
+import org.springframework.context.event.EventListener;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.stereotype.Component;
+import org.synyx.urlaubsverwaltung.person.Person;
+import org.synyx.urlaubsverwaltung.person.PersonCreatedEvent;
+import org.synyx.urlaubsverwaltung.person.PersonDeletedEvent;
+import org.synyx.urlaubsverwaltung.person.PersonDisabledEvent;
+import org.synyx.urlaubsverwaltung.person.PersonService;
+import org.synyx.urlaubsverwaltung.person.PersonUpdatedEvent;
+
+@ConditionalOnProperty(value = "uv.extensions.enabled", havingValue = "true")
```

Comment:
lass uns hierfür noch ein ConfigurationProperties einführen, sodass wir es dokumentiert haben und autocomplete in den *.properties

Reply:
was muss man zusätzlich machen, dass es dokumentiert ist? oder passiert das dann implizit durch einführen eines ConfigurationProperties?

Reply:
ja nur einführen